ts/dx.all.d.ts

```diff
@@ -2673,6 +2680,22 @@ declare module DevExpress.ui {
         /** @name dxGalleryItem.imageSrc */
         imageSrc?: string;
     }
+    /** @name dxGantt.Options */
+    export interface dxGanttOptions extends WidgetOptions<dxGantt> {
+        /** @name dxGantt.Options.dependenciesDataSource */
+        dependenciesDataSource?: any;
+        /** @name dxGantt.Options.resourceAssignmentsDataSource */
+        resourceAssignmentsDataSource?: any;
+        /** @name dxGantt.Options.resourcesDataSource */
+        resourcesDataSource?: any;
+        /** @name dxGantt.Options.taskDataSource */
+        taskDataSource?: any;
```

Comment:
Why is task prefix singular but dependencies, resourceAssignments and resources are plural

Reply:
Also, "any" type is not good for data sources.